service/build.gradle

```diff
@@ -59,6 +59,12 @@ dependencies {
     implementation group: "org.apache.commons", name: "commons-text", version: "1.9"
     annotationProcessor group: "org.springframework.boot", name: "spring-boot-configuration-processor"
 
+    // VUMC Admin dependency
+    implementation group: "bio.terra", name: "tanagra-vumc-admin-client", version: "0.0.1d-SNAPSHOT"
```

Comment:
Do you know what the longer term plan is for this yet?

Reply:
Sorry, I should have been more specific. I was thinking more about how this is integrated into the core service.

Reply:
I'm planning to add a new implementation of the `AccessControlPlugin` interface. Currently, we only have `OpenAccessControlPlugin` that allows all requests. `VumcAdminAccessControlPlugin` will call the `isAuthorized` and `listAuthorizedResources` endpoints of the admin service. And then I can use an underlay config property to specify which plugin class to call. (I'll tackle this in a future PR.)

That should work with the way things are setup now. However, once Erik starts developing the admin service, he may decide to remove those endpoints or change the service architecture, and we'd have to adapt here to those changes. e.g. One way this could change is if he wants core Tanagra to notify the admin service every time we create an artifact. In that case, we'd need to expand the access control plugin interface to also call VUMC admin service at create time. If it were my call, I would not take that approach, so that's why I held off implementing it until we know more details of Erik's plan here.